usmqe/web/tendrl/mainpage/dashboard/models.py

```diff
@@ -0,0 +1,17 @@
+"""
+Page model for dashboard
+"""
+
+
+from webstr.core import WebstrModel
+
+
+LOCATION = '/#/dashboard'
+
+
+class DashboardModel(WebstrModel):
+    """
+    Page model for dashboard
+    """
+# Coming soon...
```

Comment:
In truth I should not put here dashboard at all, as it's not part of latest build.
Not sure which issue probably https://github.com/Tendrl/dashboard/issues/120 I'll put it to the start of the models.py file